Workbooks/Communication Services/Overview/Overview.workbook

```diff
@@ -365,13 +365,7 @@
                           "durationMs": 86400000
                         },
                         "id": "2666fe6a-842f-46ad-9f75-5e52e8517ff8",
-                        "value": [
```

Comment:
Qq: Why this change ?

Reply:
Sorry, that came from the graphical editor, I've restored the default values for this parameter.